src/Symfony/Component/DependencyInjection/Tests/ContainerTest.php

```diff
@@ -285,15 +285,15 @@ public function testGetCircularReference()
     }
 
     /**
-     * @expectedException \Symfony\Component\DependencyInjection\Exception\RuntimeException
-     * @expectedExceptionMessage You have requested a synthetic service ("request"). The DIC does not know how to construct this service.
+     * @expectedException \Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException
```

Comment:
We go from `\RuntimeException` to `\InvalidArgumentException` now.. minor BC break :)

:+1: for master, as it's needed to keep things simple.